src/actions/actions.test.js

```diff
@@ -22,11 +22,71 @@ describe('Actions', () => {
   })
 
   it('should have a STORE_USER_QUOTE with a correct payload', () => {
-
+    const userQuote = { systemSize: 1, moduleType: 0, arrayType: 2, systemLosses: 14, tilt: 45, azimuth: 7 }
+    const expectedAction = {
+      type: 'STORE_USER_QUOTE',
+      userQuote
+    }
+    const result = actions.allUserQuotes(userQuote)
+      expect(result).toEqual(expectedAction)
   })
 
   it('should have a SET_SOLAR_DATA with a correct payload', () => {
+    const data = {data:
+       {ac_monthly: [521.8912353515625,
```

Comment:
For large datasets like this, since it's just for test files we might want to consider putting all values in an array on one line.  So a line for ac_monthly and all its data, a line for solrad_monthly and all its data, etc.  I will change up my data presentation to this format for consistency in our code and we can discuss this consideration later.